app/controllers/discovered_hosts_controller.rb

```diff
@@ -38,14 +38,10 @@ def show
     @range = nil
     # summary report text
     @report_summary = nil
-    init_regex_and_categories
-    @interfaces = []
-    get_interfaces
-    @host.facts_hash.each do |key, value|
-      value = number_to_human_size(value) if /size$/.match(key)
-      assign_fact_to_category(key, value)
-    end
-    add_custom_facts
+    @categories_names = [N_("Highlights"), N_("Storage"), N_("Hardware"), N_("Network"), N_("Software"), N_("IPMI"), N_("Miscellaneous")]
```

Comment:
This is once here and again in the new class, is that necessary?

Reply:
I see, It was a leftover. Now this array in only in new class